src/execute/oas3/style-serializer.js

```diff
@@ -59,10 +43,14 @@ export default function stylize(config) {
 export function valueEncoder(value, escape) {
   if (Array.isArray(value) || (value !== null && typeof value === 'object')) {
     value = JSON.stringify(value);
+  } else if (typeof value === 'number' || typeof value === 'boolean') {
+    value = value.toString();
```

Comment:
I'd suggest to use this notation instead of `toString()`, as `isString()` is called by `String` function as of of the mechanism of coercion (not the only one).
```suggestion
    value = String(value);
```